x2gbfs/providers/moqo.py

```diff
@@ -144,11 +144,13 @@ def _extract_from_parkings(
             'name': elem['name'],
             'lat': center['lat'],
             'lon': center['lng'],
-            'address': address,
-            'post_code': elem['zipcode'],
             'rental_methods': ['key'],
         }
 
+        if address:
+            station['address'] = address
```

Comment:
Are address / their components really optional? Above, you access them via [], are the properties contained but empty?

Reply:
Oberschwaben Mobil has a station "RV-Nordstadt" with geocoordinates, but without any address information. The attributes town, zipcode, street, streetnumber are available, but they are = "".